scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -254,7 +252,7 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
 
 }
 
-trait TestScalaJSModule extends ScalaJSModule with TestModule {
+trait TestScalaJSModule extends TestModule with ScalaJSModule {
```

Comment:
Is this change of order relevant? I think it shouldn't.

Reply:
Reverted ✅